_content/doc/tutorial/greetings-multiple-people.html

```diff
@@ -159,7 +159,7 @@
     names := []string{"Gladys", "Samantha", "Darrin"}</ins>
 
     // Request greeting messages for the names.
-    messages, err := greetings.Hellos(names)
+    <ins>messages, err := greetings.Hellos(names)</ins>
```

Comment:
I came here to create a PR with this same change. I would recommend making this a continuation of line 159 though, because the comment changed as well.

Reply:
Yes, you're right. Just fixed it. Thanks.